ui/src/components/CopyToClipboardMultiline.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { OverlayTrigger, Tooltip } from 'patternfly-react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+class CopyToClipboardMultiline extends Component {
+  render() {
+    return (
+      <div className="input-group" style={{ display: 'block', overflowX: 'auto', ...this.props.style }}>
```

Comment:
So far we are using classes to define our styling. We can use the patternfly spacing and colors that way.
Can you move the inline styling to scss?